DotNet/emitter.js

```diff
@@ -366,7 +389,8 @@
 				let MatchTimeout = !checkSettings.timeout ? `` : `match_timeout:${checkSettings.timeout}`
 				let Tag = !checkSettings.name ? `` : `tag:"${checkSettings.name}"`
 				if (Tag !== `` && MatchTimeout !== ``) Tag = `, ` + Tag
-				let isFully = !checkSettings.isFully ? `` : `, fully:"${checkSettings.isFully}"`
+				let isFully = (checkSettings.isFully === undefined) ? `` : `fully:${checkSettings.isFully}`
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
				let isFully = (checkSettings.isFully === undefined) ? "" : `fully:${checkSettings.isFully}`
```